packages/vuetify/src/util/mask.ts

```diff
@@ -11,35 +17,37 @@ export const defaultDelimiters = /[-!$%^&*()_+|~=`{}[\]:";'<>?,./\\ ]/
  *
  * @return {Boolean}
  */
-export const isMaskDelimiter = char => char && defaultDelimiters.test(char)
+export const isMaskDelimiter = (char: string): boolean => char ? defaultDelimiters.test(char) : false
 
 /**
  * Mask keys
  *
  * @type {Object}
  */
-const allowedMasks = {
+const allowedMasks: Record<MaskType, MaskItem> = {
   '#': {
-    test: char => char.match(/[0-9]/)
+    test: (char: string): boolean => Boolean(char.match(/[0-9]/)),
+    convert: (char: string) => char
```

Comment:
`convert` can be made optional in the interface

Reply:
If I try this, I have some problems